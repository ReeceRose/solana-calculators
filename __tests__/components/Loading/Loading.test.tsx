import { render } from '@testing-library/react';

import Loading from '../../../components/Loading/Loading';

test('Loading component matches snapshot', () => {
  const { asFragment } = render(<Loading />);
  expect(asFragment()).toMatchSnapshot();
});
