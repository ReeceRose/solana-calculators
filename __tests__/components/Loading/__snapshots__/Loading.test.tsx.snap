// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="fixed top-0 bottom-0 left-0 right-0 z-50 flex flex-col items-center justify-center w-full h-screen overflow-hidden bg-gray-700 opacity-75"
  >
    <div
      class="w-12 h-12 border-b-4 border-blue-500 rounded-full animate-spin"
    />
  </div>
</DocumentFragment>
`;
